heart_rate_imposer.py

```diff
@@ -0,0 +1,154 @@
+from data_analysis.heart_rate_generator import HeartRateGenerator
```

Comment:
delete this file, it has nothing to do with the server, it was just for testing...